src/components/HdTabsMenu.vue

```diff
@@ -54,13 +54,18 @@ export default {
       type: Boolean,
       default: false,
     },
+    // Available options: left, right, center
+    alignment: {
+      type: String,
+      default: 'left',
```

Comment:
I suggest replacing the comment with a validator instead.
```suggestion
    alignment: {
      type: String,
      default: 'left',
      validator: (value) => ['left', 'right', 'center'].includes(value),
```

Reply:
good point! @aym3nb